src/BugBounty.js

```diff
@@ -18,16 +18,16 @@ export default _.assign(
       codeHash
     ) => {
       console.log('new BugBounty')
-      const bugBounty = await BugBounty.new(
+      const BugBounty = await BugBounty.new(
```

Comment:
these need to be named differently - `BugBounty` is the contract artifact while `bugBounty` is the instance of the new contract.